docs/docs/sign-in-out.md

```diff
@@ -2,8 +2,21 @@
 title: Sign In and Out
 layout: docs
 ---
+## Signing in and out
+To sign in and out of Tableau Server, simply call `Auth.sign_in` as follows:
+```py
+import tableauserverclient as TSC
+
+tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD')
+server = TSC.Server('http://SERVER_URL')
+
+with server.auth.sign_in(tableau_auth):
+    # Do awesome things here!   
+
+# The Auth context manager will automatically call 'Auth.sign_out' on exiting the with block
```

Comment:
I reverted the change, but did shift the note below the `with` block example, since it looked weird sitting there in between